qpext/cmd/qpext/main.go

```diff
@@ -88,16 +93,25 @@ func negotiateMetricsFormat(contentType string) expfmt.Format {
 	return expfmt.FmtText
 }
 
-// scrapeAndWriteAgentMetrics gathers a slice of prometheus metric families and encodes the metrics.
-func scrapeAndWriteAgentMetrics(w io.Writer) error {
-	mfs, err := promRegistry.Gather()
-	if err != nil {
-		return err
```

Comment:
istio has agent, we don't ... so this basically did nothing but the code is useful for what we need to do now